Makefile

```diff
@@ -437,6 +437,21 @@ install_upstream: build_upstream
 	(cd _build_upstream && $(MAKE) install)
 	cp ocaml/VERSION $(prefix)/lib/ocaml/
 
+.PHONY: build_and_test_upstream
+build_and_test_upstream: build_upstream
+	if $$(which gfortran > /dev/null 2>&1); then \
```

Comment:
It looks like the CI is failing because
lines 269-270 ([here](https://github.com/ocaml-flambda/flambda-backend/blob/dd81e9ad08cdf96524c6de030d07de65439c4d96/Makefile#L269)) are missing.

Reply:
It's actually because `--enable-ocamltest` needs to be passed to `configure` or the `make`-based build system doesn't build `ocamltest`...